how-to/customize-workspace/package.json

```diff
@@ -15,14 +15,16 @@
 		"client": "npm --prefix ../common/ run client",
 		"secondclient": "npm --prefix ../common/ run secondclient",
 		"thirdclient": "npm --prefix ../common/ run thirdclient",
+		"fourthclient": "npm --prefix ../common/ run fourthclient",
 		"server": "node ./server/express/build/index.js",
 		"kill": "npm --prefix ../common/ run kill",
 		"setup": "cd ../../ && npm install && cd how-to/customize-workspace && npm run build && npm --prefix ../common/ run build",
 		"setup-dev": "npm install concurrently live-server --save-dev && npm run build-server-live",
 		"validate": "npm --prefix ../../ run validate",
 		"generate-types": "tsc --project ./client/tsconfig.types.json",
+		"generate-types-module": "tsc --project ./client/tsconfig.types-module.json",
```

Comment:
The new type generation is confusing, if you run both commands you end up with duplicate types in different folders.

Reply:
combined into one command generate types

Reply:
The output is still a bit odd, lots of duplicates, could easily lead to confusion e.g. `framework/shapes/action-shapes.d.ts` and `module/action-shapes.d.ts`

Reply:
We can address this in the intellisense pull request as it deals with types and extends this.